distributed/config.yaml

```diff
@@ -58,3 +58,5 @@ worker-memory-terminate: 0.95  # fraction at which we terminate the worker
 
 log-length: 10000  # default length of logs to keep in memory
 log-format: '%(name)s - %(levelname)s - %(message)s'
+
+client-heartbeat-interval: 5000  # milliseconds between client heartbeats
```

Comment:
I'd rather have seconds everywhere :-)

Reply:
Yeah, I was thinking about this as I was writing this.  Historically we have preferred milliseconds when we're just passing through to Tornado or Bokeh and they expect seconds.  However we also use seconds when we do our own thing.  This has plenty of opportunity for confusion.

As a solution I suggest that we make a `parse_time` function and use strings like `"5000 ms"` in configuration files in the future.  

Smoothly transitioning will be tricky no matter what solution we choose.

Reply:
> Tornado or Bokeh and they expect seconds

Should be 

Tornado or Bokeh and they expect milliseconds

Reply:
I suspect Bokeh copied Tornado's initial API mistake. But, well, no need to accumulate our own debt here :-)

Reply:
Yeah, I'm in favor of changing this.  This is a larger task though and will need to be done consistently across the codebase.

Reply:
@pitrou any objection to holding off on this change until we change it globally?  For the moment this choice matches current conventions in the codebase.

I'm happy to commit to changing to seconds or strings everywhere prior to the next 0.X.0 release.